MAUI/ImageEditor/Crop.md

```diff
@@ -0,0 +1,155 @@
+platform: MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Crop in .NET MAUI ImageEditor (SfImageEditor)
+
+You can use the cropping tool in the Image Editor to select and crop a specific portion of an image.
+
+## Image cropping ratio
+
+The Image Editor control provides various selection options for cropping images, including custom, square, circle, and customized to various aspects ratios. You can drag and resize the selection region to crop the desired area of an image.
+
+Cropping operation can be done in the following two ways:
+
+* Enable cropping and visually select the crop region.
+* Manually entering the cropping area.
+
+N> You can also crop the image at a specific position with annotations. 
+
+## Handling the cropping tool
+
+The `Crop` method in the image editor control crop the image based on the `ImageCropType`. It allows users to enable or disable the cropping region displayed over the image, making it easier to visually choose the area for cropping.
+
+    * ImageCropType - Specifies the selection type for cropping the image.
+
+* The following code shows cropping the image to any desired size.
+
+{% highlight C# %}
+
+imageEditor.Crop(ImageCropType.Free);
+
+{% endhighlight %}
+
+After the cropping area has been selected, the `SaveEdits` method is called, which in turn crops the selected region and displays the cropped image on the image editor.
+
+{% highlight C# %}
+
+imageEditor.SaveEdits();
+
+{% endhighlight %}
+
+After selecting the cropping area, if you decide to cancel the edits and revert back to the original image, you can use the `CancelEdits` method. It discard the changes made during the cropping process and restore the image to its initial state.
+
+{% highlight C# %}
+
+imageEditor.CancelEdits();
+
+{% endhighlight %}
+
+## Freehand Crop
+
+This type of crop allows you to freely select and crop a region of the image. You can manually outline the desired area using your cursor or touch input.The selection region can be customized by dragging and resizing an image.
+
+{% highlight C# %}
+
+imageEditor.Crop(ImageCropType.Free);
+
+{% endhighlight %}
+
+## Original Crop
+
+With the original crop, the image is cropped to its original dimensions. This means that no aspect ratio and the cropped image retains its original width and height.
+
+{% highlight C# %}
+
+imageEditor.Crop(ImageCropType.Original);
+
+{% endhighlight %}
+
+## Square Crop
+
+The square crop option allows you to crop the image into a perfect square shape. This is useful when you want to create square thumbnails or profile pictures.
+
+{% highlight C# %}
+
+imageEditor.Crop(1,1);
+
+{% endhighlight %}
+
+## Ratio Crop
+
+The ratio crop enables you to crop the image with a specific aspect ratio. You can specify the desired ratio using the width and height values. For example, setting the ratio to 16:9 will result in a crop with a width-to-height ratio of 16:9.
+
+{% highlight C# %}
+
+imageEditor.Crop(16,9);
```

Comment:
mention if specified ImageCropType.Ratio is used we consider default ratio 4: 3 and if you want to change the ratio you can use crop method with ratio imageEditor.Crop(16,9);